Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/callback/opensaml/HOKSAMLAssertionBuilder.java

```diff
@@ -100,6 +100,9 @@ public Element build(final CallbackProperties properties) throws SAMLAssertionBu
         LOG.debug("SamlCallbackHandler.createHOKSAMLAssertion20()");
         Element signedAssertion = null;
         try {
+            final X509Certificate certificate = certificateManager.getDefaultCertificate();
+            final PublicKey publicKey = certificateManager.getDefaultPublicKey();
+            final PrivateKey privateKey = certificateManager.getDefaultPrivateKey();
             Assertion assertion;
```

Comment:
you can group them like this Assertion assertion = OpenSAML2ComponentBuilder.getInstance().createAssertion();